content/docs/contributing/style_guide.md

```diff
@@ -30,5 +30,22 @@ Whenever possible, the handbook's prose should be written in Markdown rather tha
 However, it is completely reasonable use HTML when it is needed.
 When it is advantageous, data should be [stored in data files and processed using shortcodes](https://gohugo.io/templates/data-templates/) rather than presented in raw Markdown.
 
+## Style Enforcement
+
 Markdown styling is enforced by [markdownlint](https://github.com/markdownlint/markdownlint) using the configuration {{% repo_link path=".mdl_style.rb" text=".mdl_style.rb" %}}.
 An explanation of the rules can be found [here](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md).
+
+You can install markdownlint locally to check your changes before submitting a pull request.
+Make sure you install the Ruby version (`mdl`) and not the Node.js version (`markdownlint-cli`) contained in the homebrew repositories,
+since the configuration is only good for the former.
```

Comment:
This information is duplicated.

I think it makes more sense on this page than the getting started page.
Using the hint admonition here might help it stand out.

Reply:
Thanks; I've removed the info from the getting started page and emphasised it as a hint here as you suggested.